### PR TITLE
develop

### PR DESCRIPTION
**Pull request by panianchuk**
https://panianchuk.github.io/js_growth_table_DOM/

### CODE REVIEW

File: src/scripts/main.js [hunk 1]
```diff
@@ -1,3 +1,63 @@
...
+    : btnDelRow.disabled = false;
+
+  tbody.firstChild.children.length >= 10
+    ? btnAddCol.disabled = true
+    : btnAddCol.disabled = false;
+
+  tbody.children.length <= 2
+    ? btnDelCol.disabled = true
+    : btnDelCol.disabled = false;
+});
```
Comment (SanyaBratashchuk) [inline src/scripts/main.js:43]: move lengths to the variables

**Review by SanyaBratashchuk — CHANGES_REQUESTED** [review]

File: src/scripts/main.js [hunk 2]
```diff
@@ -1,3 +1,65 @@
...
+    addColumn(tbody.children);
+  }
+
+  if (target === btnDelCol) {
+    delColumn(tbody);
+  }
+
+  tbody.children.length >= maxNumbersBlocks
+    ? btnAddRow.disabled = true
+    : btnAddRow.disabled = false;
```
Comment (ivanchuk-yurii) [inline src/scripts/main.js:32]:
```suggestion
  btnAddRow.disabled = tbody.children.length >= maxNumbersBlocks
```

File: src/scripts/main.js [hunk 3]
```diff
@@ -1,3 +1,65 @@
...
+    delRow(tbody.children);
+  }
+
+  if (target === btnAddCol) {
+    addColumn(tbody.children);
+  }
+
+  if (target === btnDelCol) {
+    delColumn(tbody);
+  }
```
Comment (ivanchuk-yurii) [inline src/scripts/main.js:28]: it's better to use `switch case`

**Review by ivanchuk-yurii — CHANGES_REQUESTED** [review]

File: src/scripts/main.js [hunk 4]
```diff
@@ -1,3 +1,67 @@
...
+    : btnAddRow.disabled = false;
+
+  tbody.firstChild.children.length <= minNumbersBlocks
+    ? btnDelRow.disabled = true
+    : btnDelRow.disabled = false;
+
+  tbody.firstChild.children.length >= maxNumbersBlocks
+    ? btnAddCol.disabled = true
+    : btnAddCol.disabled = false;
+
```
Comment (k-zhdanova) [inline src/scripts/main.js:43]: Still not resolved for these cases 
Screenshot 2022-12-06 at 19 59 33

**Review by k-zhdanova — CHANGES_REQUESTED** [review]

**Pull request closed without merging**